.gitignore

```diff
@@ -42,8 +42,8 @@ test.*
 .attach_pid*
 
 # Environment Variables
-!.env.example
-.env*
```

Comment:
Do not change this, especially not to this.

L45 is now saying "do not ignore any ``.env``"  
L46 is now saying "Ignore any ``.env``"  
So you have made a contradiction

This relates to why did you delete the ``.env.example``

Reply:
didn't even realise I changed them sorry, will fix it now